eslint/babel-eslint-parser/src/convert/convertAST.js

```diff
@@ -23,7 +19,15 @@ function convertNodes(ast, code) {
       }
     },
     exit(path) {
-      const node = path.node;
+      const { node } = path;
+
+      if (Object.hasOwnProperty.call(node, "extra")) {
```

Comment:
Seems unlikely ESLint would need this, but please let me know if I'm wrong.

Reply:
AFAIK `node.extra` is only used internally in babel-parser. It is a collection of per-node parser features, `parenthesized`, `trailingComma`, etc. So we can safely remove `node.extra`.

Reply:
Got it, thanks!